### PR TITLE
chore(docs): update health check documentation

### LINKED ISSUE #1664: Update - Document configuration for health check pessimism
See https://github.com/questdb/questdb/pull/3292

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Closes #1664

Also documents symbol capacity tuning for COPY statement.

### CODE REVIEW

File: docs/guides/importing-data.md [hunk 1]
```diff
@@ -378,9 +371,30 @@ dd bs=1 count=1000 skip=56 if=input_file.csv 2>/dev/null | head -1 >> errors.csv
...
+
+```questdb-sql
+CREATE TABLE table_name (
+  ts TIMESTAMP,
+  sym SYMBOL CAPACITY 100000
+) TIMESTAMP(ts) PARTITION BY DAY;
+```
+
+Refer to [symbol type documentation](/docs/concept/symbol/#symbol-columns) for
+more information on configuring symbol capacity.
```
Comment (amyshwang) [inline docs/guides/importing-data.md:390]:
```suggestion
more information on configuring the symbol capacity.
```

File: docs/guides/importing-data.md [hunk 2]
```diff
@@ -378,9 +371,30 @@ dd bs=1 count=1000 skip=56 if=input_file.csv 2>/dev/null | head -1 >> errors.csv
...
+pre-creating the table before running the `COPY` command. Here is an example:
+
+```questdb-sql
+CREATE TABLE table_name (
+  ts TIMESTAMP,
+  sym SYMBOL CAPACITY 100000
+) TIMESTAMP(ts) PARTITION BY DAY;
+```
+
+Refer to [symbol type documentation](/docs/concept/symbol/#symbol-columns) for
```
Comment (amyshwang) [inline docs/guides/importing-data.md:389]:
```suggestion
Refer to the [symbol type documentation](/docs/concept/symbol/#symbol-columns) for
```

File: docs/guides/importing-data.md [hunk 3]
```diff
@@ -378,9 +371,30 @@ dd bs=1 count=1000 skip=56 if=input_file.csv 2>/dev/null | head -1 >> errors.csv
...
 
 <details>
-  <summary>What happens in a database crash or OS reboot?</summary>
+  <summary>COPY on a table with symbol columns is very slow. How can I speed it up?</summary>
 <p>
 
+QuestDB uses `256` as the default symbol capacity. In case if the number of
+distinct symbol column values exceeds this default significantly, the `COPY`
+performance will suffer. Make sure that you specify symbol capacities when
+pre-creating the table before running the `COPY` command. Here is an example:
```
Comment (amyshwang) [inline docs/guides/importing-data.md:380]:
```suggestion
performance will suffer. Make sure that you specify a symbol capacity when
creating the table before running the `COPY` command. 

Here is an example:
```

File: docs/guides/importing-data.md [hunk 4]
```diff
@@ -378,9 +371,30 @@ dd bs=1 count=1000 skip=56 if=input_file.csv 2>/dev/null | head -1 >> errors.csv
 ## FAQ
 
 <details>
-  <summary>What happens in a database crash or OS reboot?</summary>
+  <summary>COPY on a table with symbol columns is very slow. How can I speed it up?</summary>
 <p>
 
+QuestDB uses `256` as the default symbol capacity. In case if the number of
+distinct symbol column values exceeds this default significantly, the `COPY`
+performance will suffer. Make sure that you specify symbol capacities when
```
Comment (amyshwang) [inline docs/guides/importing-data.md:379]:
```suggestion
QuestDB uses `256` as the default symbol capacity. If the number of
distinct symbol values exceeds this default significantly, the `COPY`
performance will suffer. Make sure that you specify symbol capacities when
```

File: docs/guides/importing-data.md [hunk 5]
```diff
@@ -130,8 +123,8 @@ csvstack *.csv > singleFile.csv
 
 ### Configure `COPY`
 
-- Enable `COPY` and [configure](/docs/reference/configuration/#csv-import) `COPY`
-  directories to suit your server.
+- Enable `COPY` and [configure](/docs/reference/configuration/#csv-import)
+  `COPY` directories to suit your server.
```
Comment (amyshwang) [inline docs/guides/importing-data.md:127]:
```suggestion
- Enable `COPY` and [configure](/docs/reference/configuration/#csv-import)
  the `COPY` directories to suit your server.
```

**Review by amyshwang — APPROVED** [review]
Some minor tweaks.

File: docs/guides/importing-data.md (hunk 4 above)
Comment (puzpuzpuz) [reply docs/guides/importing-data.md]: Done in 700c38836a40040a09ee9c7cc7136bfa976b152e

File: docs/guides/importing-data.md (hunk 5 above)
Comment (puzpuzpuz) [reply docs/guides/importing-data.md]: Done in 700c38836a40040a09ee9c7cc7136bfa976b152e

**Review by amyshwang — COMMENTED** [review]
A question!

File: docs/reference/configuration.md [hunk 6]
```diff
@@ -243,6 +244,7 @@ default on port `9000`. For details on the use of this component, refer to the
...
 | http.security.interrupt.on.closed.connection | true         | Switch to enable termination of SQL processing if the HTTP connection is closed. The mechanism affects performance so the connection is only checked after `circuit.breaker.throttle` calls are made to the check method. The mechanism also reads from the input stream and discards it since some HTTP clients send this as a keep alive in between requests, `circuit.breaker.buffer.size` denotes the size of the buffer for this. |
+| http.pessimistic.health.check.enabled        | false        | When enabled, health check will return HTTP 500 if there were any unhandled errors since the server start.                                                                                                                                                                                                                                                                                                                             |
```
[the remark this answers is not in the document]
Comment (puzpuzpuz) [reply docs/reference/configuration.md]: These errors are unhandled and, thus, critical. They're not necessarily critical in our internal meaning.

[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/reference/configuration.md]: Thanks - I think we should align the wording - will add a commit.
I think both places could just use unhandled for simplicity.

**Review by amyshwang — APPROVED** [review]

**Pull request merged**